src/components/Inputs/TextMaskedInput/TextMaskedInput.js

```diff
@@ -60,7 +60,7 @@ export const TextMaskedInput = (props) => {
 
   const setAllTouched = () => {
     whichSetTouched(true)
-    if (!!setFieldTouched) {
+    if (setFieldTouched) {
```

Comment:
I think we actually need this since you only get this if you hit these from Form.js unfortunately. So we need to preserve all `!!setFieldTouched` parts

Reply:
I'm not sure I understand. How does the `!!` here matter?

Reply:
I was wrong on this. I just pulled `fix/lint-problems-i` and it's fine 👍